rebasehelper/tests/test_versioneer.py

```diff
@@ -42,7 +42,7 @@ class TestVersioneer(object):
     ])
     def test_anitya_versioneer(self, package, min_version):
         assert AnityaVersioneer.get_name() in versioneers_runner.versioneers
-        version = versioneers_runner.run(AnityaVersioneer.get_name(), package, None)
+        version = versioneers_runner.run(AnityaVersioneer.get_name(), package, None, [])
```

Comment:
The last argument could default to empty list, that way you wouldn't have to specify it at all.